habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/PyenvAndPoetrySetup.java

```
@@ -159,7 +158,7 @@ public void registerRepositoryToSupportAuthenticatedDependencyResolution(String
         PoetryCommandHelper poetryHelper = createPoetryCommandHelper();
 
         if (StringUtils.isEmpty(username) || StringUtils.isEmpty(password)) {
-            log.info(String.format("Did not find username and password for the server with <id> %s. Will use existing configuration.", repoId));
+            throw new HabushuException("Your credentials for " + repoId + " must be set in your ~/.m2/settings.xml");
```

I: Throwing exception to fail fast